src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -298,7 +298,9 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
                   {
                     val delete = asyncDeleteMessages(pi.partitionNr, group map (MessageId(persistenceId, _)))
                     delete.onFailure {
-                      case e => log.warning(s"Unable to complete deletes for persistence id ${persistenceId}, toSequenceNr ${toSequenceNr}. The plugin will continue to function correctly but you will need to manually delete the old messages.", e)
+                      case e => log.warning(s"Unable to complete deletes for persistence id ${persistenceId}, toSequenceNr ${toSequenceNr}. " +
```

Comment:
we should use placeholders `{}` for `persistenceId` and `toSequenceNr` also, instead of string interpolation